pkg/handlers/generic/mutation/extraapiservercertsans/inject.go

```diff
@@ -63,24 +65,36 @@ func (h *extraAPIServerCertSANsPatchHandler) Mutate(
 		h.variableFieldPath...,
 	)
 	if err != nil {
-		return err
+		log.Error(
+			err,
+			"failed to get cluster config variable from extraAPIServerCertSANs mutation handler",
+		)
 	}
 	if !found {
 		log.V(5).Info("Extra API server cert SANs variable not defined")
 	}
-	apiCertSANs := extraAPIServerCertSANsVar
+	cluster, err := clusterGetter(ctx)
+	if err != nil {
+		log.Error(
+			err,
+			"failed to get cluster from extraAPIServerCertSANs mutation handler",
+		)
+	}
+	defaultAPICertSANs := getDefaultAPIServerSANs(cluster)
+	apiCertSANs := make([]string, len(extraAPIServerCertSANsVar))
+	copy(apiCertSANs, extraAPIServerCertSANsVar)
+	apiCertSANs = append(apiCertSANs, defaultAPICertSANs...)
```

Comment:
Let's ensure that there are no duplicates in here. 

A combo of slices.Clone, slices.Sort, and slices.Compact would work nicely here.

Reply:
done